File: Windows Drive Space Manager CSharp/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace Windows_Drive_Space_Manager_CSharp
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
            DiskSpaceResources.Resources.PrepareComboTable();
            cmbDisplaySize.DataSource = DiskSpaceResources.Resources.ComboTable;
            cmbDisplaySize.DisplayMember = "DisplayLong";
            cmbDisplaySize.ValueMember = "DisplayShort";
            cmbDisplaySize.SelectedIndex = 2;
        }

        private void btnRefresh_Click(object sender, EventArgs e)
        {
            DiskSpaceResources.Resources.PrepareDiskTable(cmbDisplaySize.SelectedValue.ToString());
            dgvResults.DataSource = DiskSpaceResources.Resources.DiskTable;
        }
    }
}

File: Windows Drive Space Manager VB/Form1.vb
Public Class Form1

    Private Sub Form1_Load(sender As System.Object, e As System.EventArgs) Handles MyBase.Load
        DiskSpaceResources.Resources.PrepareComboTable()
        cmbDisplaySize.DataSource = DiskSpaceResources.Resources.ComboTable
        cmbDisplaySize.DisplayMember = "DisplayLong"
        cmbDisplaySize.ValueMember = "DisplayShort"
        cmbDisplaySize.SelectedIndex = 2
    End Sub

    Private Sub btnRefresh_Click(sender As System.Object, e As System.EventArgs) Handles btnRefresh.Click
        DiskSpaceResources.Resources.PrepareDiskTable(cmbDisplaySize.SelectedValue)
        dgvResults.DataSource = DiskSpaceResources.Resources.DiskTable
    End Sub
End Class
